queries.sql

```diff
@@ -57,3 +57,41 @@ SELECT species, MAX(weight_kg) AS maximum_weight_kg,
 SELECT species, ROUND(AVG(escape_attempts), 2) AS escape_attempts
   FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000
   GROUP BY species;
+
+-- JOIN querries
+SELECT full_name AS owner, animals.name AS animal
+  FROM owners
+  JOIN animals on animals.owners_id=owners.id
+  WHERE owners.full_name='Melody Pond';
+
+SELECT animals.name AS animal, species.name AS species
+  FROM animals
+  JOIN species on animals.species_id=species.id
+  WHERE species.name='Pokemon';
+
+SELECT full_name AS owner, animals.name AS animal
+  FROM owners
+  LEFT JOIN animals on animals.owners_id=owners.id;
+
+SELECT species.name AS species, COUNT(animals.name) AS animals
+  FROM species
+  JOIN animals on animals.species_id=species.id
+  GROUP BY species.name;
+
+SELECT owners.full_name AS owner, animals.name AS animal, species.name AS species
+  FROM owners
+  JOIN animals ON animals.owners_id=owners.id
+  JOIN species ON animals.species_id=species.id
```

Comment:
To give you a more accurate search, kindly consider using the `left join` instead of the `Join` here. 
An inner join (join) returns only the rows that have matching values in both tables being joined. If there is no match for a particular row in one of the tables, that row will not appear in the result set. It effectively filters out unmatched rows from both tables. It is more restrictive and only includes rows that have corresponding matches in both tables.
On the other hand, A left join returns all rows from the left (or first) table and the matching rows from the right (or second) table. If there is no match for a particular row in the right table, the result will still include the row from the left table, with NULL values for columns from the right table. It ensures that all rows from the left table are included in the result set, even if there are no matching rows in the right table. :pray:

Reply:
Hello @jerryowusu thank you for pointing this out. However, in this particular instance I wanted to match values from all three tables. The prompt was to **list all Digimon owned by Jennifer Orwell**. The owner's name, the name of the animal, and the type of the animal were all in three separate tables. While there are cases where a left join is the preferred method, I don't think it would have produced better results for THIS specific prompt seeing as the double-joins gave the exact response that was required.